src/platform-integration/web/wasm.md

```diff
@@ -3,45 +3,53 @@ title: Support for WebAssembly (Wasm)
 description: >-
   Current status of Flutter's experimental support for WebAssembly (Wasm).
 short-title: Wasm
-last-update: November 21, 2023
+last-update: March 8, 2024
 ---
 
+**_Last updated {{page.last-update}}_**
+
 The Flutter and Dart teams are excited to add
 [WebAssembly](https://webassembly.org/) as a compilation target when building
 applications for the web.
 
-{{site.alert.warning}}
-  Development of WebAssembly support for Dart and Flutter remains ongoing.
-  
-  You can preview Wasm and WebAssembly garbage collection (WasmGC)
-  on the [`master` channel][].
-  Flutter is previewing these features.
-  Ongoing development might result in frequent changes.
-  Revisit this page for the latest updates.
-  
-  **_Last updated {{page.last-update}}_**
+Development of WebAssembly support for Dart and Flutter remains ongoing,
+which will potentially result in frequent changes. 
+Revisit this page for the latest updates.
+
+{{site.alert.note}}
+  **Wasm is now in beta!**
+  : Wasm and WebAssembly garbage collection (WasmGC) are now 
+    available on the Flutter [`beta` channel][] and [`master` channel][].
 {{site.alert.end}}
 
-[`master` channel]: https://github.com/flutter/flutter/wiki/flutter-build-release-channels#master
+{{site.alert.note}}
+  **Dart's next-gen Web interop is now stable!**
+  : Migrate your packages to [`package:web`][] and [`dart:js_interop`][]
+    to make them compatible with Wasm. Read the
+    [Requires JS-interop](#requires-js-interop-to-access-browser-and-js-apis)
+    section to learn more. 
+{{site.alert.end}}
+
+[`beta` channel]: https://github.com/flutter/flutter/wiki/flutter-build-release-channels#beta
+[`master` channel]: https://github.com/flutter/flutter/wiki/flutter-build-release-channels#beta
+[`package:web`]: {{site.pub-pkg}}/web
+[`dart:js_interop`]: {{site.dart.api}}/{{site.dart.sdk.channel}}/dart-js_interop 
 
 ### Background
 
-To compile Dart and Flutter to Wasm, you need
-a browser that supports [WasmGC][].
+To compile Dart and Flutter to Wasm, you need a browser that supports [WasmGC][].
```

Comment:
I know this text existed already, but technically a WasmGC browser is not required to compile, but rather to run it.

Maybe something like:

```suggestion
To run a Flutter app that has been compiled to Wasm,
you need a browser that supports [WasmGC][].
```